docs/_data/extra_pages.json

```diff
@@ -0,0 +1,10 @@
+[
+    { "category": "Project", "title": "Brand", "url": "https://brand.systemd.io/" },
+    { "category": "Project", "title": "Releases", "url": "https://github.com/systemd/systemd/releases" },
+    { "category": "Project", "title": "GitHub Project Page", "url": "https://github.com/systemd/systemd" },
+    { "category": "Project", "title": "Issues", "url": "https://github.com/systemd/systemd/issues" },
+    { "category": "Project", "title": "Pull Requests", "url": "https://github.com/systemd/systemd/pulls" },
+    { "category": "Project", "title": "Mailing List", "url": "https://lists.freedesktop.org/mailman/listinfo/systemd-devel" },
+    { "category": "Manual Pages", "title": "Index", "url": "https://www.freedesktop.org/software/systemd/man/" },
+    { "category": "Manual Pages", "title": "Directives", "url": "https://www.freedesktop.org/software/systemd/man/systemd.directives.html" }
+]
```

Comment:
This could be a good place to mention systemd-stable, maybe ?

Reply:
hmm, i figure that would better be a document of its own explaining the situation. A link about this without context (i.e. how it is managed, when it is rebased, who manages it) seems only medium helpful to me... Hence, maybe put together another .md file that explains the situation.

Reply:
(which I think should be a separate PR, which I'd be happy to review)